src/Mime_ext.jl

```diff
@@ -2,7 +2,7 @@ module Mime_ext
 
 export MimeExt
 
-MimeExt = Dict{AbstractString,AbstractString}(
+MimeExt = Dict{String,String}(
```

Comment:
Unrelated, but I wonder why this isn't `const`.

Reply:
Good point, let's Make It So.